easybuild/framework/easyblock.py

```diff
@@ -106,8 +106,6 @@
 from easybuild.tools.version import this_is_easybuild, VERBOSE_VERSION, VERSION
 
 
-EASYBUILD_SOURCES_URL = 'https://sources.easybuild.io'
```

Comment:
We can indeed define it in `easybuild.tools.config`, but then importing it in `easybuild.framework.easyblock` and not using it makes houndbot complain. I defined it separately in two locations to fix houndbot, but it's not optimal.

Reply:
@joeydumont Hmm, I would be surprised if the bot complains about a constant not being used in `easybuild.tools.config`, but if it does, you can add a comment to skip the style check for that line:

```python
EASYBUILD_SOURCES_URL = 'https://sources.easybuild.io'  # noqa

Reply:
I meant in `easybuild.framework.easyblock`, if we had
```
from easybuild.tools.config import EASYBUILD_SOURCES_URL
```
so that it doesn't break the API. That's why I've done now, but added `# noqa`. 

In short, now `EASYBUILD_SOURCES_URL` is defined in `easybuild.tools.config`, where it is used, and imported in `easybuild.framework.easyblock`, where it used to be defined, so it can still be imported as `from easybuild.framework.easyblock`, but defined in a single place. The `# noqa` is required for HoundBot not to complain about the unused variable in `easybuild.framework.easyblock`.

Reply:
:+1: